Headers/state_diag_mod.F90

```diff
@@ -174,7 +174,7 @@ MODULE State_Diag_Mod
      LOGICAL :: Archive_Loss
      LOGICAL :: Archive_Prod
 
-#ifdef MODEL_GEOS
+!#ifdef MODEL_GEOS
```

Comment:
This change includes adding fields beyond what is needed for CH4 pseudo flux diagnostics. Is that intentional?